tutor/resources/images/highlighter/instructions-panel-icon.svg

```diff
@@ -0,0 +1,31 @@
+                            <path d="M0.5,7 C0.223857625,7 3.38176876e-17,6.77614237 0,6.5 L0,0.5 C-3.38176876e-17,0.223857625 0.223857625,5.07265313e-17 0.5,0 L10.5,0 C10.7761424,-5.07265313e-17 11,0.223857625 11,0.5 L11,6.5 C11,6.77614237 10.7761424,7 10.5,7 L8.5,7 L7.5,8 L6.5,7 L0.5,7 Z" id="Combined-Shape" fill="#FFFFFF"></path>
+                            <rect id="Rectangle" fill="#0DC0DC" x="1.5" y="3.5" width="6" height="1" rx="0.2"></rect>
+                            <rect id="Rectangle-Copy" fill="#0DC0DC" x="1.5" y="1.5" width="8" height="1" rx="0.2"></rect>
+                        </g>
+                        <g id="noun_highlighter_99741_ffffff-2" transform="translate(7.000000, 7.500000)">
+                            <g id="noun_highlighter_99741_ffffff">
+                                <path d="M7.75623932,0 C6.8839886,0 6.32387464,0.455313393 6.32387464,0.455313393 C6.23031339,0.531372134 6.0754416,0.654084393 5.97982906,0.727857216 L2.48159544,3.42825424 C2.38586895,3.50213097 2.30769231,3.65601485 2.30769231,3.77031077 L2.30769231,4.54211996 C2.30769231,4.65641588 2.37960114,4.81653408 2.46757835,4.8979959 L4.71190883,6.97693483 C4.79988604,7.05839665 4.97435897,7.125 5.0997151,7.125 L5.92603989,7.125 C6.05139601,7.125 6.21903134,7.05268185 6.29846154,6.96436228 L9.21447293,3.72313772 C9.29401709,3.63471424 9.42678063,3.49225996 9.50951567,3.40643411 C9.50951567,3.40643411 10,2.89802507 10,2.07800378 C9.99988604,0.930368809 8.9954416,0 7.75623932,0 Z M5.51270655,5.34375 L4.87168091,4.74993073 L7.43589744,2.37496536 L8.07680912,2.96868073 L5.51270655,5.34375 Z" id="Shape" fill="#FFFFFF" fill-rule="nonzero"></path>
+                                <polygon id="Path" fill="#0DC0DC" points="5.51270655 5.34375 4.87168091 4.74993073 7.43589744 2.37496536 8.07680912 2.96868073"></polygon>
+                                <path d="M3.78838821,7.30523305 C3.86540906,7.37237086 3.86540906,7.48224051 3.78838821,7.54937832 L3.50714239,7.79463685 C3.43012154,7.86177466 3.27737682,7.91666667 3.16763209,7.91666667 L0.19953588,7.91666667 C0.0897911458,7.91666667 0,7.8395952 0,7.74539675 L0,7.10930026 C0,7.0151018 0.06295357,6.88313833 0.13997442,6.81600052 L1.54360956,5.59202002 C1.62063041,5.52488221 1.74653755,5.52488221 1.8235584,5.59202002 L3.78838821,7.30523305 Z" id="Path" fill="#FFFFFF" fill-rule="nonzero"></path>
+                                <path d="M9.44444444,8.91666667 L0.555555556,8.91666667 C0.248777778,8.91666667 0,8.69276667 0,8.41666667 C0,8.14056667 0.248777778,7.91666667 0.555555556,7.91666667 L9.44444444,7.91666667 C9.75122222,7.91666667 10,8.14056667 10,8.41666667 C10,8.69276667 9.75122222,8.91666667 9.44444444,8.91666667 Z" id="Path" fill="#0DC0DC" fill-rule="nonzero"></path>
+                            </g>
+                        </g>
+                    </g>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
This is fine, but just so you're aware, there's 2 ways we do icons and we haven't really settled on the best approach yet.  

The other method is to convert the SVG to a react component using https://www.smooth-code.com/open-source/svgr/playground/ or one of the other convertors out there.

That has the advantage of playing nicely with styled-components and lets you alter the SVG properties using react props if needed.

Reply:
That's interesting, I wonder about the tradeoffs with asset caching.